crates/uv/tests/pip_install_scenarios.rs

```diff
@@ -1941,7 +1941,7 @@ fn local_greater_than() {
 
     ----- stderr -----
       × No solution found when resolving dependencies:
-      ╰─▶ Because only package-a<=1.2.3 is available and you require package-a>1.2.3, we can conclude that the requirements are unsatisfiable.
+      ╰─▶ Because only package-a==1.2.3+foo is available and you require package-a>1.2.3, we can conclude that the requirements are unsatisfiable.
```

Comment:
Here we know there is exactly one version available, so we show that instead of a range.